.github/workflows/build.yml

```diff
@@ -10,7 +10,8 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
+  build-core-blockchain:
+    name: build-core-blockchain
     runs-on: ubuntu-20.04
     steps:
     - name: Install latest nightly
```

Comment:
Should be just named `Install Rust toolchain`. Same to the other jobs.

Reply:
Done.